driver/bpf/types.h

```diff
@@ -220,8 +220,9 @@ enum scap_map_types {
 	SCAP_TMP_SCRATCH_MAP = 7,
 	SCAP_SETTINGS_MAP = 8,
 	SCAP_LOCAL_STATE_MAP = 9,
+	SCAP_INTERESTING_SYSCALLS_TABLE = 10,
```

Comment:
`SCAP_STASH_MAP` should always stay last because it can be missing when `BPF_SUPPORTS_RAW_TRACEPOINTS` is enabled, therefore all below idx would be shifted by one.

Reply:
Hmm how didn't we have this table before?

Reply:
We passed a sentinel value while filling the syscall_table map instead; see old impl of populate_syscall_table_map() in scap_bpf.c